src/main/java/api/App.java

```diff
@@ -75,17 +70,27 @@ public static void run() {
                         + "<a href=\"https://github.com/A1Liu/schedge\">"
                         + "check out the repository</a>.";
 
-      var info =
-          new Info().version("0.1").title("Schedge").description(description);
-
-      config.enableWebjars();
-
-      var options = new OpenApiOptions(info)
-                        .path("/api/swagger.json")
-                        .reDoc(new ReDocOptions("/api"));
-      config.registerPlugin(new OpenApiPlugin(options));
-
-      config.addStaticFiles(staticFiles -> {
+      var info = new OpenApiInfo();
+      info.setVersion("2.0.0");
+      info.setTitle("Schedge");
+      info.setDescription(description);
+
+      config.staticFiles.enableWebjars();
+
+      var htmlPath = "/api";
+      var jsonPath = "/api/swagger.json";
+      var openApiConfig = new OpenApiConfiguration();
+      openApiConfig.setInfo(info);
+      openApiConfig.setDocumentationPath(jsonPath);
+      var reDocConfig = new ReDocConfiguration();
+      reDocConfig.setDocumentationPath(jsonPath);
+      reDocConfig.setWebJarPath("/api/webjars");
+      reDocConfig.setUiPath(htmlPath);
+      config.plugins.register(new OpenApiPlugin(openApiConfig));
+      config.plugins.register(new ReDocPlugin(reDocConfig));
+
+      config.staticFiles.add(staticFiles -> {
+        // Not working anymore
         staticFiles.hostedPath = "/";
         staticFiles.directory = "/next/";
         staticFiles.location = Location.CLASSPATH;
```

Comment:
From what I understand, this is the same as it was before, but somehow it isn't working anymore.